server/src/main/java/org/opensearch/index/compositeindex/datacube/startree/aggregators/MaxValueAggregator.java

```diff
@@ -65,11 +65,16 @@ public Long toLongValue(Double value) {
     }
 
     @Override
-    public Double toStarTreeNumericTypeValue(Long value, StarTreeNumericType type) {
+    public Double toStarTreeNumericTypeValue(Long value) {
         try {
-            return type.getDoubleValue(value);
+            return VALUE_AGGREGATOR_TYPE.getDoubleValue(value);
         } catch (Exception e) {
             throw new IllegalStateException("Cannot convert " + value + " to sortable aggregation type", e);
         }
     }
+
+    @Override
+    public long getIdentityMetricValue() {
+        return Long.MIN_VALUE;
```

Comment:
Just adding so that we don't miss it.

This should be null - lets verify the defaults as discussed offline and address here. Applicable for all aggregators.

```
value_count = 0
sum = 0.0
min = null
max = null
avg = null
```